text/0043-phrase-query.md

```diff
@@ -0,0 +1,161 @@
+- Title: Phrase Query
+- Start Date: 2021-06-02
+- Specification PR: [#43](https://github.com/meilisearch/specifications/pull/43)
+- MeiliSearch Tracking-Issues:
+
+# Phrase Query
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Meilisearch does not allow users a way to write a strict query in order to ask the engine to be more drastic in its selection of candidates for search results. The Phrase Query feature adds a simple syntax available to users to require the engine to select documents that will strictly match the search field value. That is, without typography, without prefix and, in the order of the given terms.
```

Comment:
```suggestion
MeiliSearch does not allow users a way to write a strict query in order to ask the engine to be more drastic in its selection of candidates for search results. The Phrase Query feature adds a simple syntax available to users to require the engine to select documents that will strictly match the search field value. That is, without typography, without prefix and, in the order of the given terms.
```

Reply:
Also, without n-gram (gather two words) and split words (trying to split the word into 2 other words)